scripts/configure_delivery.sh

```diff
@@ -29,34 +29,34 @@ echo "$(tput bold)Setting up Cloud Build triggers...$(tput sgr0)"
 echo
 
 # Check input env variables
-if [[ -z "${PROD_PROJECT}" ]]; then
+if [ -z "${PROD_PROJECT}" ]; then
     echo "Please set the $(tput bold)PROD_PROJECT$(tput sgr0) variable"
     exit 1
-elif [[ -z "${STAGE_PROJECT}" ]]; then
+elif [ -z "${STAGE_PROJECT}" ]; then
     echo "Please set the $(tput bold)STAGE_PROJECT$(tput sgr0) variable"
     exit 1
-elif [[ -z "${OPS_PROJECT}" ]]; then
+elif [ -z "${OPS_PROJECT}" ]; then
     echo "Please set the $(tput bold)OPS_PROJECT$(tput sgr0) variable"
     exit 1
 fi
 
 REPO_CONNECT_URL="https://console.cloud.google.com/cloud-build/triggers/connect?project=${OPS_PROJECT}"
 echo "Connect your repos: ${REPO_CONNECT_URL}"
-read -rp "Once your forked emblem repo is connected, please continue by typing any key."
+read -rp "Once your forked emblem repo is connected, please continue by typing any key." EMPTY_VAR
 
-if [[ -z "${REPO_NAME}" || -z "${REPO_NAME}" ]]
```

Comment:
I think we should keep the non-posix compliant double brackets, but yes this line should be changed to remove the second part of this OR

Reply:
Done